packages/documentation/src/content/docs/playground/autopeering.md

```diff
@@ -23,6 +23,8 @@ To use the [Bruno API examples](/playground/overview/#bruno--open-payments-apis)
 3. copy the url of one of the wallet addresses
 4. set the url as `senderWalletAddress` variable in the Bruno `Autopeering` environment
```

Comment:
After the steps we should say something like now run the Open Payments example folder
(since only that would work for out of the box)

Reply:
See L19